src/components/UI/Buttons/Buttons.tsx

```diff
@@ -4,16 +4,50 @@ import styles from './Buttons.module.sass';
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
   onClick: () => void;
   children: JSX.Element | string;
+  icon?: string; // Can't put here JSX.Element due to clash with JSX.HTMLAttributes
+  isReversed?: boolean;
+  iconAlt?: string;
 }
```

Comment:
- Add to the `children` prop an option to assign multiple elements.
- Remove the `icon`, `isReversed` and `iconAlt` props, can be implemented directly from parent element.
```suggestion
interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
  onClick: () => void;
  children: JSX.Element | JSX.Element[] | string;
}
```

Reply:
Fixed this mess